### ISSUE TITLE
Terminal accessibility help should have top/bottom padding

### ISSUE THREAD

**Issue by Tyriar** [microsoft/vscode#172408]
<img width="1160" alt="image" src="https://user-images.githubusercontent.com/2193314/214620016-df3c762b-c414-4cd5-a3a1-ff41e4d118e3.png">

Instead of setting the height explicitly, try something like this?

```css
max-width: 80%;
overflow: scroll;
transform: translate(-50%, -50%);
left: 50%;
top: 50%;
```
